src/Model/Behavior/RegisterBehavior.php

```diff
@@ -31,11 +31,11 @@ class RegisterBehavior extends BaseTokenBehavior
     /**
      * @var bool
      */
-    protected $validateEmail;
+    protected bool $validateEmail = true;
```

Comment:
The default value would be the one from config. 

We have ` $this->validateEmail = (bool)Configure::read('Users.Email.validate');`  

And since the default value for the config `Users.Email.validate` is true it make sense to set true here.